sqlmesh/core/model/kind.py

```diff
@@ -53,6 +53,7 @@ def is_incremental(self) -> bool:
             self.is_incremental_by_time_range
             or self.is_incremental_by_unique_key
             or self.is_incremental_unmanaged
+            or self.is_scd_type_2
```

Comment:
Can you explain why this change was made? SCD Type 2 doesn't inherit from the `_Incremental` model kind for example. What aspect of it makes it incremental in order to be added here?

Reply:
Because this kind is conceptually incremental. Today we only use this method (`is_incremental`) when we check for data gaps. I think the check should apply to scd type 2 as well.

Reply:
Want to change this to something like `can_be_partiall_loaded` or `allows_loading_subset`?

Reply:
what's wrong with `is_incremental`?